templates/Dockerfile.txt

```diff
@@ -1,8 +1,8 @@
-FROM microsoft/aspnet:1.0.0-beta8
-
-EXPOSE 5000
-ENTRYPOINT ["dnx", "-p", "project.json", "web"]
+FROM microsoft/aspnet:1.0.0-rc1
 
 COPY . /app
 WORKDIR /app
-RUN ["dnu", "restore"]
\ No newline at end of file
+RUN ["dnu", "restore"]
+
+EXPOSE 5000
```

Comment:
Please change this to `EXPOSE 5000/tcp` per convention and consistency
@ahmetalpbalkan submitted a PR #471 - please bring this part in.